packages/fxa-admin-panel/package.json

```diff
@@ -8,10 +8,11 @@
     "restart": "pm2 restart pm2.config.js",
     "lint:eslint": "eslint .",
     "build": "npm-run-all build:client build:server",
-    "build:client": "react-scripts build",
+    "build:client": "rescripts build",
     "build:server": "tsc -p server/tsconfig.json",
     "test": "npm-run-all test:*",
-    "test:frontend": "react-scripts test --coverage --verbose",
+    "pretest:frontend": "cd ../fxa-components && npm install",
```

Comment:
This isn't ideal, but is required to make testing work. Why? Because our tests render components that import shared components, and those shared components need to reference their own dependencies.

If anyone has a suggestion for how to improve this, I'm all ears, considering it needs to be added to any project that references imported components.